webapp/src/components/ask_ai_input.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import styled from 'styled-components';
+
+import IconAI from './assets/icon_ai';
+
+const Container = styled.div`
+	display: flex
+	position: relative
+	border: 1px solid #ddd
+	borderRadius: 5px
+	padding: 5px
+	right: 7px
+`;
+
+const Input = styled.input`
+	border: none
+	backgroundColor: transparent
+`;
+
+export default function AskAiInput() {
```

Comment:
I am guessing this will be used in a follow up PR?

Reply:
Yes